src/Cosi-BLS/cosi-handlers.lisp

```diff
@@ -1316,10 +1316,12 @@ bother factoring it with NODE-COSI-SIGNING."
 
 ;; -------------------------------------------------------------
 
+#|
```

Comment:
@guitarvydas Just go ahead and remove such references to a specific network configuration.